### DISCUSSION TITLE
amazing app, well done

### DISCUSSION THREAD

**Discussion by bmw216** [xanderfrangos/twinkle-tray#865] [General]
i would like to congratulate the person/s behind this app. im amazed how configurable is it for free.

thank u team twinkle tray 🧡

**Comment by xanderfrangos** [comment]
Thank you! I'm basically the only one on the "team", but the app wouldn't be as good nor as popular without the many Node packages used, translators who have donated their time, and users who provide valuable feedback and kind words like yours. 😊
